packages/fx-core/src/plugins/resource/bot/questions.ts

```diff
@@ -38,22 +39,22 @@ const botIdQuestion = new QTreeNode({
 
 const botPasswordQuestion = new QTreeNode({
   name: QuestionNames.GET_BOT_PASSWORD,
-  type: NodeType.password,
+  type: "text",
+  password: true,
   title: "Enter bot password",
   default: "",
   validation: {
-    validFunc: async (botPassword: string) => {
-      if (!botPassword) {
+    validFunc: async (input: string, previousInputs?: Inputs) => {
+      if (!(input as string)) {
```

Comment:
same here for as.
BTW, the para name (input) is too general. Keep its previous name?